download.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: download
+title: "Download - Jamulus"
+lang: "en"
+permalink: /download
+
+---
+
+
+    <div class="container">
+        <div class="fx-col-100-xs fx-col-40-m fx-center infoarea">
+            <h3>Got a pair of headphones and an instrument/mic?<br>If so, you're basically all set!</h3>
+            <div class="requirements">
+                <p>Minimum System Requirements</p>
+                <ul class="list">
+                    <li class="listitem">Stable Internet Connection</li>
```

Comment:
```suggestion
                    <li class="listitem">Stable Internet Connection (WiFi isn't stable; Use Ethernet)</li>
```

Reply:
How about: "Stable Internet Connection (Ethernet recommended)

Reply:
I think we should leave min requirements as brief as possible

Reply:
If that's clear enough, sure.